test/integration/targets/nxos_smoke/tests/common/common_config.yaml

```diff
@@ -0,0 +1,158 @@
+---
+# nxos_config -> NetworkConfig, dumps
+# nxos_static_route -> CustomNetworkConfig
+
+# hit NetworkConfig
+# Select interface for test
+- debug: msg="START connection={{ connection }} nxos_smoke common_config test"
```

Comment:
How about using this debug statement:
```
- debug: msg="START connection={{ ansible_connection }}/set_hostname.yaml"
- debug: msg="Using provider={{ connection.transport }}"
  when: ansible_connection == "local"
```
This displays which connection plugin it is using and which transport explicitly.